doc/api/cluster.markdown

```diff
@@ -357,18 +357,19 @@ This example will echo back all messages from the master:
       });
     }
 
-### worker.suicide
+### worker.exitedVoluntarily (formerly worker.suicide)
 
 * {Boolean}
 
 Set by calling `.kill()` or `.disconnect()`, until then it is `undefined`.
```

Comment:
Nit: If you fix the comma splice below, might as well fix this one too? Change comma to period/full-stop and have `until then` be the start of a new sentence. (Bonus: Put a comma after `then`: "Until then, it is `undefined`."